packages/wonder-blocks-grid/util/utils.js

```diff
@@ -63,5 +63,6 @@ export const flexBasis = (size: number | string) => {
 };
 
 export const gridContextTypes = {
-    gridSize: propTypes.oneOf(["small", "medium", "large"]),
```

Comment:
I removed this as the type wasn't being checked at all and it seemed unnecessary. This will get rewritten when we upgrade to React 16+ anyway.

Reply:
Why do we need manual `propTypes` here?  Can't we use `flow` types for this?

Reply:
Ha, we can't :D - Flow explicitly can't be used with context types. This whole thing will go away once we start using React 16, though.